nixops/backends/none.py

```diff
@@ -32,10 +34,11 @@ class NoneState(MachineState):
     def get_type(cls):
         return "none"
 
+    provison_ssh_key: bool = nixops.util.attr_property("provisionSSHKey", None, bool)
```

Comment:
```suggestion
    provison_ssh_key: bool = nixops.util.attr_property("provisionSSHKey", true, bool)
```

and I wonder about making the attr_property return type be validated against its default